src/WikisourceApi.php

```diff
@@ -104,15 +104,24 @@ public function fetchWikisources( $cacheLifetime = null ) {
 		if ( $data === false ) {
 			$this->logger->debug( "Requesting list of Wikisources from Wikidata" );
 			$query =
-				"SELECT ?item ?langCode ?langName WHERE { "
+				"SELECT ?item ?langCode ?langName WHERE { {"
 				// Instance of Wikisource language edition.
 				. "?item wdt:P31 wd:Q15156455 . "
 				// Wikimedia language code.
 				. "?item wdt:P424 ?langCode . "
 				// Language of work or name.
 				. "?item wdt:P407 ?lang . "
 				// RDF label of the language, in the language.
-				. "?lang rdfs:label ?langName . FILTER(LANG(?langName) = ?langCode || ?langCode = 'mul') . " . "}";
+				. "?lang rdfs:label ?langName . FILTER(LANG(?langName) = ?langCode) ."
+				// perform a union of the results
+				. " } UNION { "
+				// Instance of Wikisource language edition.
```

Comment:
You could add `BIND( "Multilingual Wikisource" AS ?langName )` to hard-code the language name. That'd fix the warning below and make things slightly simpler.